### PR TITLE
add template param to getToken; update copy regarding tokens/jwt templates

### PR DESCRIPTION
**Pull request by alexisintech**
[DOCS-474](https://linear.app/clerk/issue/DOCS-474/feedback-for-referencesbackendsessionsget-token) mentions that the `getToken()` method requires a `template` param.

This issue was brought up to Team SDK in [Slack](https://clerkinc.slack.com/archives/C046VRJPCNS/p1697568510249879) as this parameter should be optional. A [ticket in Linear](https://linear.app/clerk/issue/SDK-830/update-template-param-of-gettoken-backend-api-to-be-optional) was created to update this param.

For now, this PR updates the docs to include the `template` param. 
This PR also updates the explanation for the `template` param where necessary, expanding on how to specify the string passed to `template`.
This PR also updates the default claims section of the [JWT Templates page](https://clerk.com/docs/backend-requests/making/jwt-templates#default-claims) to be consistent with the default claims section of the [Session Tokens page](https://clerk.com/docs/backend-requests/resources/session-tokens#default-session-claims).

### CODE REVIEW

File: docs/backend-requests/making/jwt-templates.mdx [hunk 1]
```diff
@@ -57,15 +57,27 @@ A token generated using the template above, would look roughly like this:
...
+* `exp`: expiration time - the time after which the token will expire, as a Unix timestamp. Determined using the **Token lifetime** template setting in the [Clerk Dashboard](https://dashboard.clerk.com/last-active?path=jwt-templates).
+* `iat`: issued at - the time at which the token was issued, as a Unix timestamp (e.g. `1516239022`).
+* `iss`: issuer - the Frontend API URL of your instance (e.g. `https://clerk.your-site.com`).
+* `nbf`: not before - the time before which the token is considered invalid, as a Unix timestamp. Determined using the **Allowed Clock Skew** template setting in the [Clerk Dashboard](https://dashboard.clerk.com/last-active?path=jwt-templates).
+* `sid`: session ID - the ID of the current session.
+* `sub`: subject - the ID of the current user of the session (e.g. `user_123`).
+* `orgs`: organizations - the organizations that the user belongs to.
+* `org_id`: organization ID - the ID of the organization that the user belongs to.
+* `org_slug`: organization slug - the slug of the organization that the user belongs to.
+* `org_role`: organization role - the role of the user in the organization.
```
Comment (kostaspt) [inline docs/backend-requests/making/jwt-templates.mdx:70]: While `org_x` exists in session JWT tokens, they don't in JWT Templates. Also, `orgs` is deprecated, so this won't be returned for new applications in either.

File: docs/backend-requests/making/jwt-templates.mdx [hunk 2]
```diff
@@ -57,15 +57,27 @@ A token generated using the template above, would look roughly like this:
 
 In the previous example, "automatically-included claims" was briefly mentioned. Besides the claims specified by the template, there are certain claims that are automatically included in all tokens and cannot be overridden by templates. Clerk calls these "default claims" and they are the following:
 
-- `sub`: The user's unique ID (e.g. `user_abc1234def57`)
-- `iat`: The time at which the token was issued, as a Unix timestamp (e.g. `1516239022`)
-- `iss`: The Frontend API URL of your instance (e.g. `https://clerk.your-site.com`)
-- `jti`: A random alphanumeric string, used as a unique identifier for the token
```
Comment (kostaspt) [inline docs/backend-requests/making/jwt-templates.mdx:63]: `jti` needs to be included below, as it's still accurate.

File: docs/backend-requests/resources/session-tokens.mdx [hunk 3]
```diff
@@ -11,13 +11,13 @@ When a user is authenticated in your application, a short-lived session token is
 
 Below are the default session claims that Clerk generates for you:
 
-* `azp`: authorized party - the `Origin` header that was included in the original Frontend API request made from the user. Most commonly, it will be the URL of the application, e.g. `https://example.com`.
-* `exp`: expiration time - the time at which this token expires.
-* `iat`: issued at - the time at which this token was issued.
-* `iss`: issuer - the Frontend API URL of the instance, e.g. `https://clerk.example.com`.
-* `nbf`: not before - the time before which this token cannot be used.
+* `azp`: authorized party - the `Origin` header that was included in the original Frontend API request made from the user. Most commonly, it will be the URL of the application (e.g. `https://example.com`).
```
Comment (kostaspt) [inline docs/backend-requests/resources/session-tokens.mdx:14]: `azp` could be omitted if for privacy-related reasons we get as `Origin` empty or `null`.

File: docs/backend-requests/resources/session-tokens.mdx [hunk 4]
```diff
@@ -11,13 +11,13 @@ When a user is authenticated in your application, a short-lived session token is
...
+* `exp`: expiration time - the time after which the token will expire, as a Unix timestamp. Determined using the **Token lifetime** template setting in the [Clerk Dashboard](https://dashboard.clerk.com/last-active?path=jwt-templates).
+* `iat`: issued at - the time at which the token was issued, as a Unix timestamp (e.g. `1516239022`).
+* `iss`: issuer - the Frontend API URL of your instance (e.g. `https://clerk.your-site.com`).
+* `nbf`: not before - the time before which the token is considered invalid, as a Unix timestamp. Determined using the **Allowed Clock Skew** template setting in the [Clerk Dashboard](https://dashboard.clerk.com/last-active?path=jwt-templates).
 * `sid`: session ID - the ID of the current session.
-* `sub`: subject - the ID of the current user of the session.
+* `sub`: subject - the ID of the current user of the session (e.g. `user_123`).
 * `orgs`: organizations - the organizations that the user belongs to.
```
Comment (kostaspt) [inline docs/backend-requests/resources/session-tokens.mdx:21]: `orgs` is deprecated, so this won't be returned for new applications.

**Review by kostaspt — COMMENTED** [review]
Left some comments regarding the claims for JWT templates and session tokens.

File: docs/backend-requests/resources/session-tokens.mdx (hunk 4 above)
Comment (rachelnabors) [reply docs/backend-requests/resources/session-tokens.mdx:21]: How do we indicate deprecations in documentation currently?

Comment (kostaspt) [reply docs/backend-requests/resources/session-tokens.mdx:21]: Not really sure. I was thinking of either putting that into the text description like `[Deprecated]` or just moving it to another section below, indicating `**Deprecated**`, and having just the `orgs` as an item for now. We could even remove it as it's about instances created more than a year ago.

File: docs/backend-requests/making/jwt-templates.mdx [hunk 5]
```diff
@@ -57,24 +57,23 @@ A token generated using the template above, would look roughly like this:
...
-* `sid`: session ID - the ID of the current session.
 * `sub`: subject - the ID of the current user of the session (e.g. `user_123`).
-* `orgs`: organizations - the organizations that the user belongs to.
-* `org_id`: organization ID - the ID of the organization that the user belongs to.
-* `org_slug`: organization slug - the slug of the organization that the user belongs to.
-* `org_role`: organization role - the role of the user in the organization.
+
+The `act` claim will only be included if the user is impersonating another user:
+
 * `act`: actor - will include the ID of the impersonating user. See [user impersonation](/docs/custom-flows/user-impersonation#jwt-claims) for more information.
```
Comment (kostaspt) [inline docs/backend-requests/making/jwt-templates.mdx:70]: Oops missed that before. `act` is also session-specific and not available by default in JWT template.

**Review by kostaspt — APPROVED** [review]
LGTM 🚀

**Review by JacobMGEvans — APPROVED** [review]

**Review by rachelnabors — APPROVED** [review]

**Pull request merged**